src/bootlint.js

```diff
@@ -905,6 +905,35 @@ var LocationIndex = _location.LocationIndex;
             reporter('`.carousel-inner` must have exactly one `.item.active` child.', innersWithWrongActiveItems);
         }
     });
+    addLinter("E042", function lintFormControlOnWrongControl($, reporter) {
+        var formControlsOnWrongTags = $('.form-control:not(input):not(textarea):not(select)');
```

Comment:
Oh yeah, I always forget that that works. Let me change it then.
(There are a few other lint checks that unnecessarily use multiple `:not`s too, fyi)